doc/conf.py

```diff
@@ -305,6 +305,7 @@ def setup(app):
     'matplotlib': ('https://matplotlib.org/', None),
     'mayavi': ('http://docs.enthought.com/mayavi/mayavi', None),
     'sklearn': ('http://scikit-learn.org/stable', None),
+    'sphinx': ('http://www.sphinx-doc.org/en/stable', None),
```

Comment:
could we add a comment above here that says something to the effect of "If you have intersphinx enabled, these lines are unnecessary"?

Reply:
This is the standard `intersphinx_mapping` dict. I've already removed the unnecessary lines below